apps/ima/index.js

```diff
@@ -33,9 +33,38 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/exception',
       backLink: 'current-progress'
     },
+    '/exception': {
+      behaviours: SaveFormSession,
+      fields: [
+        'does-exception-apply',
+        'does-exception-apply-detail'
+      ],
+      template: 'does-exception-apply',
+      locals: { showSaveAndExit: true },
+      next: '/threatened-life-or-liberty'
```

Comment:
Can you add a comment that this will need to be updated once the correct next step is added

Reply:
Added in the comment for this step.